docs/content/policy-reference.md

```diff
@@ -272,6 +272,8 @@ complex types.
 | <span class="opa-keep-it-together">``z := x / y``</span>   | ``z`` is the quotient of ``x`` and ``y``  | ✅ |
 | <span class="opa-keep-it-together">``z := x % y``</span>   | ``z`` is the remainder from the division of ``x`` and ``y``  | ✅ |
 | <span class="opa-keep-it-together">``output := round(x)``</span>    | ``output`` is ``x`` rounded to the nearest integer | ✅ |
+| <span class="opa-keep-it-together">``output := ceil(x)``</span>    | ``output`` is ``x`` rounded up to the nearest integer | ✅ |
+| <span class="opa-keep-it-together">``output := floor(x)``</span>    | ``output`` is ``x`` rounded down the nearest integer | ✅ |
```

Comment:
Note that the WASM support isn't actually there yet - I'm hoping someone can help me with that before next release though ;) 

CC @srenatus @koponen-styra

Reply:
Sure thing. But let's keep the docs inline with the code base? 😉 We can tackle these in follow-up PRs, no need to block this.

Reply:
Sure, I can change this - I take it "SDK-dependent" is the politically correct way of saying "No"? 😄 I don't see any other option listed in the docs for any function.

Reply:
Not a lie anymore 🎉